.github/workflows/pull_request_checks.yml

```diff
@@ -0,0 +1,19 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize, reopened]
+
+jobs:
+  unit_test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16
+        cache: 'yarn'
```

Comment:
We should consider migrating to npm for consistency.

Reply:
Make sense.